bundles/org.eclipse.mita.base/src/org/eclipse/mita/base/util/BaseUtils.java

```diff
@@ -445,6 +445,14 @@ private static void prependNamedElementName(final StringBuilder id, final EObjec
 		Object _eGet = container.eGet(TypesPackage.Literals.NAMED_ELEMENT__NAME);
 		String name = ((String) _eGet);
 		if ((name != null)) {
```

Comment:
is that double `(())` on purpose?

Reply:
Since BaseUtils is used by something in .xcore it needed to be .java instead of .xtend. This is autogenerated code, will fix it though.